### PR TITLE
Add scroll content feature

### PR DESCRIPTION
**Pull request by Klynger**
#### What problem is this solving?

As title says

#### How should this be manually tested?

Go to the [workspace](https://modalscrollcontent--storecomponents.myvtex.com/) and click in the both rich texts:
Mobile support should be working too
image

#### Checklist/Reminders

- [x] Updated `README.md`.
- [ ] Updated `CHANGELOG.md`.
- [ ] Linked this PR to a Clubhouse story (if applicable).
- [ ] Updated/created tests (important for bug fixes).
- [ ] Deleted the workspace after merging this PR (if applicable).

#### Type of changes

✔️ | Type of Change
---|---
_ | Bug fix 
✔️ | New feature 
_ | Breaking change 
_ | Technical improvements

### CODE REVIEW

File: react/components/BaseModal.tsx [hunk 1]
```diff
@@ -23,15 +23,23 @@ const styles: Record<string, React.CSSProperties> = {
...
     keepMounted = false,
     ...rest
   } = props
 
+  useEffect(() => {
+    if (open) {
+      document.body.classList.add('overflow-y-hidden')
+    } else {
+      document.body.classList.remove('overflow-y-hidden')
+    }
```
Comment (kaisermann) [inline react/components/BaseModal.tsx]: You can use `classList.toggle` with a second conditional argument to add or remove a class: `document.body.classList.toggle('overflow-y-hidden', open)`

Comment (kaisermann) [reply react/components/BaseModal.tsx]: (thinking it through, it doesn't work on IE 10/11)

File: react/components/ModalContent.tsx [hunk 2]
```diff
@@ -1,20 +1,39 @@
...
+    {
+      ['ph5 pb5 pt5']: !dividers,
+      ['pv5 ph7 bt b--muted-3']: dividers,
+    }
+  )
+
   return (
-    <div className={`${handles.contentContainer} ${className} ph5 pb5 pt5`}>
+    <div className={classes}>
+      {/* {[... new Array(50)].map(() => children)} */}
```
Comment (jgfidelis) [inline react/components/ModalContent.tsx]: erase

File: react/components/BaseModal.tsx [hunk 3]
```diff
@@ -47,11 +55,16 @@ const BaseModal: React.FC<Props> = props => {
...
     <Portal container={container}>
-      <div {...rest} style={styles.container} onClick={handleClick} role="presentation">
+      <div
+        {...rest}
+        role="presentation"
+        onClick={handleClick}
+        style={styles.container}
+      >
+        {children}
         {backdrop === BackdropMode.none ? null : (
```
Comment (kaisermann) [inline react/components/BaseModal.tsx]:
```suggestion
        {backdrop !== BackdropMode.none && (
```

File: react/components/ModalContent.tsx [hunk 4]
```diff
@@ -1,20 +1,39 @@
 import React from 'react'
+import styles from '../styles.css'
+import classnames from 'classnames'
```
Comment (kaisermann) [inline react/components/ModalContent.tsx]: Update your `eslint-config-vtex` via `yarn upgrade`. Import order is not being automatically fixed.

File: react/components/ModalContent.tsx (hunk 2 above)
Comment (kaisermann) [inline react/components/ModalContent.tsx]: 👀

Comment (victorhmp) [inline react/components/ModalContent.tsx]: Will you use this?

File: react/styles.css [hunk 5]
```diff
@@ -15,6 +8,21 @@
...
+.paperScrollContent {
+  max-height: calc(100% - 4rem);
+}
+
+/* We have to apply margin: 2rem instead of using ma7 because if the user
+* change the value of ma7 the max-height of .paperScrollContent will break
+*/
+.paperNotFullScreen {
+  margin: 2rem;
+}
```
Comment (victorhmp) [inline react/styles.css]: This looks a bit dangerous ahahhaha

**Pull request merged**